packages/xo-server/src/xo-mixins/proxies.js

```diff
@@ -16,6 +16,10 @@ import patch from '../patch'
 import readChunk from '../_readStreamChunk'
 import { generateToken } from '../utils'
 
+const IP_REG = /^\d+\/ip$/
+const extractIp = networks =>
+  find(networks, (ip, network) => IP_REG.test(network))
```

Comment:
What if there are multiple entries?

Reply:
We choose one.

Reply:
Do you think that's the best approach?

What about IPv6?

Reply:
According to this [topic](https://discussions.citrix.com/topic/351974-xenserver-guest-metrics-ip-address/). This field not contains an `IPv6`.

Reply:
You're right `/ip` does not contain an IPv6, but we should support IPv6 if the proxy appliance does not have an IPv4.